### PR TITLE
Added two new tests to Archive & Unarchive locations

### PR DESCRIPTION
**Pull request by nsaxena03**
As part of fix #5, we have to add the tests to for the following workflows:

- Archive & Unarchive Locations 
- Download & Upload Web Users

Creating a PR for review for Archive & Unarchive location tests added.

### CODE REVIEW

File: HQSmokeTests/testPages/users/org_structure_page.py [hunk 1]
```diff
@@ -196,3 +205,24 @@ def cleanup_location(self):
...
+    def archive_location(self):
+        self.wait_to_click(self.org_menu_link_text)
+        active_loc = self.get_text(self.archive_this_loc)
+        self.wait_to_click(self.archive_buttton)
+        self.wait_to_click(self.archive_button_popup)
+        self.is_present_and_displayed(self.archive_success_message)
+        self.wait_to_click(self.show_arhcived_locations)
+        self.is_present_and_displayed(self.archive_this_loc)
+        archived_loc = self.get_text(self.archive_this_loc)
+        assert archived_loc == active_loc, "Location is not Archived"
```
Comment (kiran-tripathy) [inline HQSmokeTests/testPages/users/org_structure_page.py:218]: What are we checking here, the text of the location archived shouldn't be changed after being archived?

It's a good check to add, but then we should be marking this test case as pass if:
1. the location is present on the archived page
 This is checked here I believe ` self.is_present_and_displayed(self.archive_this_loc)`
2. NOT present on the landing page2. 
But do we check this?

Comment (kiran-tripathy) [inline HQSmokeTests/testPages/users/org_structure_page.py:218]: What are we checking here, the text of the location archived shouldn't be changed after being archived?

It's a good check to add, but then we should be marking this test case as pass if:
1. the location is present on the archived page
 This is checked here I believe ` self.is_present_and_displayed(self.archive_this_loc)`
2. NOT present on the landing page2. 
But do we check this?

File: HQSmokeTests/testPages/users/org_structure_page.py [hunk 2]
```diff
@@ -196,3 +205,24 @@ def cleanup_location(self):
...
+
+    def unarchive_location(self):
+        self.wait_to_click(self.org_menu_link_text)
+        self.wait_to_click(self.show_arhcived_locations)
+        archived_loc =self.get_text(self.archive_this_loc)
+        self.wait_to_click(self.unarchive_button)
+        self.wait_to_click(self.show_active_locations)
+        self.is_present_and_displayed(self.archive_this_loc)
+        unarchived_loc = self.get_text(self.archive_this_loc)
+        assert unarchived_loc == archived_loc, "Location not unarchived successfully"
```
Comment (kiran-tripathy) [inline HQSmokeTests/testPages/users/org_structure_page.py:228]: Same, we should pass when vice-versa of [this](https://github.com/dimagi/dimagi-qa/pull/109/files#r897706390) is true.

Comment (kiran-tripathy) [inline HQSmokeTests/testPages/users/org_structure_page.py:228]: Same, we should pass when vice-versa of [this](https://github.com/dimagi/dimagi-qa/pull/109/files#r897706390) is true.

**Review by kiran-tripathy — CHANGES_REQUESTED** [review]
I've suggested some changes, once they're addressed we should be good. Also a quick check- Has this been tested both on prod and staging?

**Comment by nsaxena03** [comment]
Hi @kiran-tripathy , I have added the remaining following tests 

- Download/ Upload Web Users
- Creation of Dedupe Cases rule
- Upload and Download Data Dictionary 

And have also made changes to the Archive and Unarchvie locations as per your feedback.

**Review by kiran-tripathy — APPROVED** [review]

**Comment by kiran-tripathy** [comment]
I added Kankana to review this, just skimmed over -- If works on CI as well, we can merge this.

**Pull request merged**